.gitignore

```diff
@@ -1 +1,2 @@
-.o
+*.o
+lda
```

Comment:
it gets really annoying to keep seeing the rebuilt executable as a changed file in git....